Rename `iterations_per_second` to `physics_ticks_per_second`

Calinou opened the pull request:
This makes it clearer that this property is only about physics FPS, not rendering FPS.

The `physics_fps` project setting was also renamed to `physics_ticks_per_second` for consistency.

See https://github.com/godotengine/godot/issues/16863#issuecomment-554682554.

akien-mga commented on core/engine.cpp:
```diff
@@ -36,12 +36,12 @@
 
-void Engine::set_iterations_per_second(int p_ips) {
+void Engine::set_physics_fps(int p_ips) {
 	ERR_FAIL_COND_MSG(p_ips <= 0, "Engine iterations per second must be greater than 0.");
```
Would be good to update this error too. And maybe rename `ips` and `p_ips` too?

lawnjelly commented:
On the other hand it could be an opportunity to replace the word frame with tick or iteration for 4.x when referring to physics, and standardising terminology.

https://en.wikipedia.org/wiki/Film_frame

The use of the term 'frame' can be, and is, a constant source of confusion (both for beginners, and in terms of explaining concepts) in that the word 'frame' is more normally associated with a rendered frame. It has been bastardised by some to refer to a physics iteration. Whereas a physics tick / iteration need not bear any resemblance to a rendered frame, and indeed explicitly doesn't in Godot. So we end up having to explain that a 'frame' is not a 'frame'...

A physics frame would more often be a physics frame of reference, which is completely different:
https://lmgtfy.com/?q=physics+frame

Without this distinction you have to qualify the term whenever used, and it becomes 'render frame' or 'physics frame' which is far more verbose.

Calinou commented:
@lawnjelly I like the idea of using the term "tick" consistently, but I'll wait for others to approve first.

aaronfranke commented:
I like the idea of changing physics "frames" to another terminology. What about calling this property `physics_tickrate`?

Side note: ideally, try to avoid referencing the issue in the commit itself, it causes GitHub to post many messages in the issue:

![Screenshot from 2020-09-20 18-24-18](https://user-images.githubusercontent.com/1646875/93723682-c0552000-fb6e-11ea-85b8-62597d858843.png)

Calinou commented:
If we go with the tick terminology, should we rename it to `physics_ticks_per_second`, `physics_tickrate` or `physics_tick_rate`?

Calinou committed:
Rename `iterations_per_second` to `physics_ticks_per_second`

This makes it clearer that this property is only about physics FPS,
not rendering FPS.

The `physics_fps` project setting was also renamed to
`physics_ticks_per_second` for consistency.

Calinou commented:
I've remade this pull request after discussing it with @pouleyKetchoupp.

pouleyKetchoupp reviewed (approved):
Looks great!

akien-mga commented:
Thanks!
